build/Dockerfile

```diff
@@ -121,7 +123,7 @@ CMD /bin/bash
 # STAGE: devnet-base
 #
 # This stage installs Foundry.
-FROM ${BASE_IMAGE} as devnet-base
+FROM golang:${GO_VERSION}-bookworm as devnet-base
```

Comment:
I think it is strange that the devnet-base, which will be used as the base for the final image, has to Go.
This should be installed in the devnet-builder image only.
Or use the Go in the base for the devnet-builder, and install the foundry twice, one time in the builder and another in the devnet image.

Reply:
>Or use the Go in the base for the devnet-builder, and install the foundry twice, one time in the builder and another in the devnet image.

I thought about using this option but eventually decided for using the golang image as base but I agree we can do it as you said.

Reply:
Yes, it is a bit strange. I really though about doing what you suggested and installing foundry twice. However, installing Go requires downloading the right binaries for the target platform, which seemed a b it convoluted at first. That's why I decided to go the way it is. But, anyway, let me see how it goes this other way.

Reply:
Changes made at ee968dc7